packages/api/__tests__/Business.test.ts

```diff
@@ -131,5 +131,70 @@ describe('@freshbooks/api', () => {
 				})
 			)
 		})
+
+		test('Verify parsed JSON -> model transform', async () => {
+			const json = `{
+                "id": 77128,
+                "name": "BillSpring",
+                "account_id": null,
+                "address": {
+                    "id": 74595,
+                    "street": "123",
+                    "city": "Toronto",
+                    "province": "Ontario",
+                    "country": "Canada",
+                    "postal_code": "A1B2C3"
+                },
+                "phone_number": null,
+                "business_clients": [
+                    {
+                        "id": 22347,
+                        "business_id": 77128,
+                        "account_id": "Xr82w",
+                        "userid": 74353,
+                        "client_business": {
+                            "business_id": 77128
+                        },
+                        "account_business": {
+                            "account_business_id": 363103,
+                            "account_id": "Xr82w"
+                        }
+                    }
+                ]
+            }`
```

Comment:
Never been in this repo before but this looks like a weird indentation? Not sure 🤓

Reply:
Yeah, it does. I'll take a look. I did have husky run on commit. I'll check manually as well.

Reply:
It looks good I think, I compared it with other data in the file. Seems like that's the way it's formatted. :)